google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/util/Plugins.java

```diff
@@ -20,6 +20,9 @@
 import com.intellij.ide.plugins.PluginManager;
 import com.intellij.openapi.extensions.PluginId;
 
+/**
+ * Helper methods for dealing with installed plugins.
```

Comment:
My comment on this may have been lost in the PR migration, but it seems like `PluginInfoService` might be a better place for this.